app/templates/components/footer-main.hbs

```diff
@@ -46,32 +46,32 @@
         <strong class="item">{{t 'Connect with us'}}</strong>
         {{#if socialLinks.supportUrl}}
           <a class="item" href="{{socialLinks.supportUrl}}" target="_blank" rel="noopener noreferrer">
-            <i class="info icon"></i> {{t 'Support'}}
+            <i class="info icon" style="float:left;margin-left:55px;"></i><span class="align-text">Support</span>
```

Comment:
Please avoid inline styling. You should do it in stylesheets that are defined in the project.